modules/yieldmoBidAdapter.js

```diff
@@ -317,10 +321,17 @@ function isMraid() {
   return !!(window.mraid);
 }
 
-function getPubcId(request) {
-  let pubcid;
-  if (request && request.userId && request.userId.pubcid && typeof request.userId === 'object') {
-    pubcid = request.userId.pubcid;
+/**
+ * Gets an id from the userId object if it exists
+ * @param {*} request 
+ * @param {*} idType 
+ * @returns an id if there is one, or undefined
+ */
+function getId(request, idType) {
+  let id;
+  if (request && request.userId && request.userId[idType] && typeof request.userId === 'object') {
+    console.log(JSON.stringify(request.userId));
```

Comment:
Please remove the `console.log`, if you need logging use the logging methods in utils.

Reply:
Ah, missed that one. Removed.